app/controllers/event_instances_controller.rb

```diff
@@ -15,7 +15,7 @@ def update
       SlackService.post_yt_link(event_instance) if (slack_notify && event_instance.yt_video_id?) || yt_video_id_changed
 
       TwitterService.tweet_hangout_notification(event_instance) if event_instance.started? && hangout_url_changed
-      TwitterService.tweet_yt_link(event_instance) if yt_video_id_changed
+      TwitterService.tweet_yt_link(event_instance) if !event_instance.youtube_tweet_sent
```

Comment:
and do we even need the logic check here if we are also checking the same logic inside `tweet_yt_link` ?